app/views/documents/index.html.erb

```diff
@@ -1,39 +1,89 @@
 <% content_for :title, t("documents.index.title") %>
 
+<%= render "govuk_publishing_components/components/button", {
+  text: "New document", href: new_document_path, margin_bottom: true
+} %>
+
 <div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-third">
+    <%= render "documents/index/filters" %>
+
+    <br/>
+    <hr/>
+    <br/>
+
+    <div class="govuk-body">
+      <strong> <%= t("documents.index.guidance.title") %></strong>
+      <br/>
+      <%= link_to "Planning, writing and managing content",
+        "https://www.gov.uk/topic/government-digital-guidance/content-publishing", class: "govuk-link" %>
+    </div>
+  </div>
+
   <div class="govuk-grid-column-two-thirds">
-    <%= render "govuk_publishing_components/components/button", {
-      text: "New document", href: new_document_path, margin_bottom: true
-    } %>
-
-    <%= render "govuk_publishing_components/components/document_list", {
-      items: @documents.map do |document|
-        {
-          link: {
-            text: document.title.presence || t("documents.index.search_results.untitled"),
-            path: document_path(document),
-            description: document.summary,
-            context: document.locale
-          },
-          metadata: {
-            public_updated_at: document.updated_at,
-            document_type: document.document_type_schema.label
-          }
-        }
-      end
-    } %>
+    <p class="govuk-body">
+      <%= t("documents.index.search_results.summary_html", count: @documents.total_count) %>
+    </p>
+
+    <% if @documents.none? %>
+      <%= govspeak_to_html t("documents.index.search_results.guidance") %>
+    <% else %>
+      <table class="govuk-table">
```

Comment:
Consider extracting this into a partial for readability (less indentation & easier to understand the `if` statement).

Reply:
Done :-).